recipes/spacerextractor/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "spacerextractor" %}
+  {% set version = "0.9.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://code.jgi.doe.gov/SRoux/{{ name }}/-/archive/{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e656f294cba47e3522214375c07d2b71e7cefc400554550b201503843f9d4532
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m flit install --deps none
+  run_exports:
+    - {{ pin_subpackage('spacerextractor', max_pin="x") }}
+
+requirements:
+  build:
+    - python >=3.8,<3.12
```

Comment:
no need to restrict the python env

Reply:
Thanks ! Both the license info and this specification should be fixed now.